docs/01_nodeos/02_usage/03_development-environment/index.md

```diff
@@ -17,6 +17,12 @@ While this option can technically be used for smart contract development, it may
 * [Configure Nodeos as a Local Two-Node Testnet](01_local-multi-node-testnet.md)
 * [Configure Nodeos as a Local 21-Node Testnet](https://github.com/EOSIO/eos/blob/master/tutorials/bios-boot-tutorial/README.md)
 
+## Block.one’s official EOSIO Testnet
+
+Test on fully-featured EOSIO stable releases with a browser based GUI on a multi-node network.
+
+* [The official EOSIO Testnet](https://testnet.eos.io/)
```

Comment:
To avoid redundancy with heading, please change to just: "[testnet.eos.io](https://testnet.eos.io/)"